### PR TITLE
feat(Reports): add today filter in view

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- add today filter in period dropdown filter
- remove for now daily visits graphic with this filter
- remove hours and daily visits graphic with this filter
- change dateTo to pass to datahub the hours of today

image

### CODE REVIEW

**Comment by andresmoschini** [comment]
image

En este caso si está incluido.

@fcoronelmakingsense que opinás si antes de mergear este PR mergeás un componente para renderizar correctamente los rangos de fecha?

**Comment by fcoronelmakingsense** [comment]
> image
> 
> En este caso si está incluido.
> 
> @fcoronelmakingsense que opinás si antes de mergear este PR mergeás un componente para renderizar correctamente los rangos de fecha?

si tenes razon, como ayer no creamos la tarea se me paso. ahora la creo y dejo este en stand by

**Comment by fcoronelmakingsense** [comment]
Add in reports box only the date from today. Maybe we can discuss about the content of the period we can put there. What do you think @mfantinims @andresmoschini  

image

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -27,22 +27,24 @@ const Reports = ({ dependencies: { datahubClient } }) => {
   const [state, setState] = useState({
     periodSelectedDays: periodSelectedDaysDefault,
     dateFrom: addDays(today, periodSelectedDaysDefault * -1),
-    dateTo: today,
+    dateTo: periodSelectedDaysDefault ? today : new Date(),
+    isTodaySelected: !periodSelectedDaysDefault,
```
Comment (andresmoschini) [inline src/components/Reports/Reports.js]: I would use something more general, like `dailyView`or something like that, I would think _today view_ as an special case of a _daily view_.

File: src/components/Reports/Reports.js [hunk 2]
```diff
@@ -27,22 +27,24 @@ const Reports = ({ dependencies: { datahubClient } }) => {
...
   });
 
   const changeDomain = async (name) => {
     const domainFound = state.domains.find((item) => item.name === name);
     setState((prevState) => ({ ...prevState, domainSelected: domainFound }));
   };
 
-  const changePeriod = (days) => {
+  const changePeriod = (daysToFilter) => {
```
Comment (andresmoschini) [inline src/components/Reports/Reports.js]: I think that in this case, `daysToFilter` is too much overloaded. 

What do you think about receiving `dateFrom` and `dateTo` parameters?

```suggestion
  const changePeriod = (dateFrom, dateTo, isDailyView) => {
```

File: src/components/shared/FormattedDateRangeText/FormattedDateRangeText.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { FormattedMessage, FormattedDate } from 'react-intl';
+import { addDays } from '../../../utils';
+
+export const FormattedDateRangeText = ({ dateFrom, dateTo, today }) => {
+  return (
+    <div>
+      {today ? (
+        <FormattedDate value={dateTo} />
```
Comment (andresmoschini) [inline src/components/shared/FormattedDateRangeText/FormattedDateRangeText.js]: Maybe it should be _dateFrom_ (which is a pure date)

```suggestion
        <FormattedDate value={dateFrom} />
```

File: src/i18n/en.js [hunk 4]
```diff
@@ -173,6 +173,7 @@ export default {
     pages: `Pages`,
     rank_time: `Time period`,
     title: `Track users behavior, analyze it and optimize your Marketing actions`,
+    today: `today`,
```
Comment (mfantinims) [inline src/i18n/en.js]: Please capitalize: Today

File: src/i18n/es.js [hunk 5]
```diff
@@ -173,6 +173,7 @@ export default {
     pages: `Página`,
     rank_time: `Período de tiempo analizado`,
     title: `Analiza el comportamiento de los usuarios y optimiza tu estrategia`,
+    today: `hoy`,
```
Comment (mfantinims) [inline src/i18n/es.js]: Please capitalize: Hoy

File: src/components/Reports/Reports.js (hunk 2 above)
Comment (andresmoschini) [reply src/components/Reports/Reports.js]: Estuvimos hablando y por la estructura del código no tiene sentido hacer eso. Esta función es la encargada de convertir la selección del combo en los valores correspondientes.

File: src/i18n/es.js (hunk 5 above)
Comment (fcoronelmakingsense) [reply src/i18n/es.js]: fixed

File: src/components/shared/FormattedDateRangeText/FormattedDateRangeText.js (hunk 3 above)
Comment (fcoronelmakingsense) [reply src/components/shared/FormattedDateRangeText/FormattedDateRangeText.js]: fixed

File: src/components/Reports/Reports.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/Reports.js]: changed

File: src/components/Reports/Reports.js [hunk 6]
```diff
@@ -15,7 +15,8 @@ import { Helmet } from 'react-helmet';
 import { Loading } from '../Loading/Loading';
 import { addDays, getStartOfDate } from '../../utils';
 
-const periodSelectedDaysDefault = 7;
+// This value means the today date
```
Comment (andresmoschini) [inline src/components/Reports/Reports.js]: Thanks!

**Pull request merged**